test/Serialization/basic_sil.swift

```diff
@@ -9,8 +9,6 @@
 // RUN: %target-build-swift -Xfrontend %clang-importer-sdk -emit-module -Xfrontend -disable-diagnostic-passes -force-single-frontend-invocation -Xfrontend -sil-serialize-all -o %t/def_basic.swiftmodule %S/Inputs/def_basic.sil
 // RUN: %target-build-swift -Xfrontend %clang-importer-sdk -emit-silgen -Xfrontend -sil-link-all -I %t %s | FileCheck -check-prefix=CHECK_DECL %S/Inputs/def_basic.sil
```

Comment:
Could you remove `-Xfrontend %clang-importer-sdk` and  `-I %S/../Inputs/clang-importer-sdk/swift-modules` everywhere in this file?  They are not needed anymore.